core.js

```diff
@@ -620,7 +620,7 @@ exports.init = function (sbot, config) {
     )
   }
 
-  function create(opts, cb) {
+  async function create(opts, cb) {
```

Comment:
Since create is not called very often it's probably fine to do this one as async, I just prefer to have them only in tests normally.

Reply:
Yeah thought that was the case, just beware of the dreaded promises :dragon_face: